lib/node_modules/@stdlib/stats/meanors/examples/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var uniform = require( '@stdlib/random/array/uniform' );
+var getDType = require( '@stdlib/ndarray/dtype' );
+var ndarray2array = require( '@stdlib/ndarray/to-array' );
+var ndarray = require( '@stdlib/ndarray/ctor' );
+var meanors = require( './../lib' );
+
+// Generate an array of random numbers:
+var xbuf = uniform( 25, 0.0, 20.0 );
```

Comment:
@Orthodox-64 Same comment.

Reply:
In general, for the top-level APIs, prefer using other high-level (i.e., ndarray) APIs in examples (e.g., `random/uniform`, `blas/ext/linspace`, etc).

Reply:
@Orthodox-64 Which previous APIs? Are you referring to `stats/base/ndarray` or top-level `stats/*` packages?